src/index.ts

```diff
@@ -201,6 +222,16 @@ function persistedOperationFromPayload(
     const hashFromPayload = options.hashFromPayload || defaultHashFromPayload;
     const hash = hashFromPayload(payload);
     if (typeof hash !== "string") {
+      const allowUnpersistedOperations =
+        options.allowUnpersistedOperation || (() => false);
+
+      if (allowUnpersistedOperations(payload)) {
```

Comment:
The payload being passed here is not the `req` object; it should be. (We should also pass the payload as a second option.)

The line `"postgraphile:httpParamsList"(paramsList, { options }) {` below I think can get access to `req`: `"postgraphile:httpParamsList"(paramsList, { options, req }) {`

Reply:
I have adapted the code to use the real `req` object, is it OK. 

Also, I'm not sure if it works with websocket too (`"postgraphile:ws:onOperation"(paramsList, { options, req }) {`)